source/common/quic/active_quic_listener.cc

```diff
@@ -46,6 +46,7 @@ ActiveQuicListener::ActiveQuicListener(
       kernel_worker_routing_(kernel_worker_routing) {
   // This flag fix a QUICHE issue which may crash Envoy during connection close.
   SetQuicReloadableFlag(quic_single_ack_in_packet2, true);
+  SetQuicFlag(FLAGS_quic_header_size_limit_includes_overhead, false);
```

Comment:
this is going to be permanent right? Let's comment what it does.

Reply:
Yes, this will be permanent to align with nghttp2.